package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "arc.js-scripts",
+  "version": "0.0.0-alpha.0",
+  "description": "scripts that run against @daostack/arc.js",
+  "scripts": {
+    "clean": "rimraf build/**/*",
+    "build": "npm run clean && tsc",
+    "help": "node ./build/scripts/arcScript.js -h",
+    "script": "node ./build/scripts/arcScript.js",
+    "ganache": "npm explore @daostack/arc.js -- npm start ganache",
+    "migrateContracts": "npm explore @daostack/arc.js -- npm start migrateContracts",
+    "createGenesisDao": "npm explore @daostack/arc.js -- npm start createGenesisDao"
+  },
+  "bin": {
+    "arcScript": "./scripts/arcScript.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dkent600/arc.js-scripts.git"
+  },
+  "keywords": [
+    "DAOstack",
+    "Arc.js",
+    "Arc",
+    "Ethereum",
+    "blockchain",
+    "DAO",
+    "javascript",
+    "smart",
+    "contracts"
+  ],
+  "author": "Doug Kent",
```

Comment:
please change that to daostack.io

Reply:
done

Reply:
https://github.com/daostack/dx-deployment-script/pull/1/commits/4a606d620bab8af0d3923dbc470b75b855f15642